google/cloud/aiplatform/models.py

```diff
@@ -1075,6 +1115,7 @@ def __init__(
         project: Optional[str] = None,
         location: Optional[str] = None,
         credentials: Optional[auth_credentials.Credentials] = None,
+        encryption_spec_key_name: Optional[str] = None,
```

Comment:
Why does this need encryption spec passed in?

Reply:
I think I left this from an earlier version, removing.